sevsnpmeasure/cli.py

```diff
@@ -23,7 +23,7 @@ def main() -> int:
                                      description='Calculate AMD SEV/SEV-ES/SEV-SNP guest launch measurement')
     parser.add_argument('--version', action='version', version=f'%(prog)s {VERSION}')
     parser.add_argument('-v', '--verbose', action='store_true')
-    parser.add_argument('--mode', choices=['sev', 'seves', 'snp'], help='Guest mode', required=True)
+    parser.add_argument('--mode', choices=['sev', 'seves', 'snp', 'ovmf-hash'], help='Guest mode', required=True)
```

Comment:
It's a bit strange to add this to the `--mode` argument. What do you think about introducing a new argument `--only-produce-ovmf-hash` , and in that case, verify that mode is `snp` ?

Reply:
I had that originally and found it a lot more clumsy, because now you have a mandatory `mode` parameter which isn't actually mandatory because you're really only generating half the digest.

What if we change this to `snp-ovmf-hash` and change the description of the feature in a way that explains SNP digest generation as a 2 step process. This mode only executes the first step. The `snp` mode can ingest the first step as input.

Reply:
Even better: While changing it accordingly, I realized we can call the mode `snp:ovmf-hash`. That makes it obvious that we're executing part of the snp hash calculation.